iOS/JSQMessagesViewController/component/component.yaml

```diff
@@ -2,7 +2,7 @@
 ---
 name: JSQMessagesViewController
 id: jsqmessagesviewcontroller
-version: 7.3.4
+version: 7.3.5

```

Comment:
This file needs to have the is_shell set and to reference the nuget that is specified in the spec.

Also need to move the licence.md file to the root folder from the components folder

This is an example -> https://github.com/newky2k/XamarinComponents/blob/dh_InAppSettingsKit/iOS/InAppSettingsKit/component/component.yaml#L22-L24